src/libse/Common/SsaStyle.cs

```diff
@@ -28,6 +28,8 @@ public class SsaStyle
         public string RawLine { get; set; }
         public bool LoadedFromHeader { get; set; }
 
+        public const string DefaultAssStyleFormat = "Format: Name, Fontname, Fontsize, PrimaryColour, SecondaryColour, OutlineColour, BackColour, Bold, Italic, Underline, StrikeOut, ScaleX, ScaleY, Spacing, Angle, BorderStyle, Outline, Shadow, Alignment, MarginL, MarginR, MarginV, Encoding";
```

Comment:
Should/could this be used more places?

Reply:
Yes, I used it in my categories PR as well. :D